packages/botonic-plugin-hubtype-analytics/src/event-models/ht-event-webview-step.ts

```diff
@@ -0,0 +1,22 @@
+import { EventAction, EventType, EventWebviewStep, RequestData } from '../types'
+import { HtEvent } from './ht-event'
+
+export class HtEventWebviewStep extends HtEvent {
+  action: EventAction.WebviewStep | EventAction.WebviewEnd
```

Comment:
Doubt: Is this `EventAction.WebviewEnd` right? Shouldn't it be just `EventAction.WebviewStep`?

Reply:
You're right